CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Changelog for NeoFS Node
 ### Added
 
 ### Fixed
+- Morph client dies when the first endpoint is unavailable even if there are more endpoints to try (#2703)
```

Comment:
can be confusing: is it about what was fixed or after changes? `from now` / `no more` help

and `Morph client` -> `NeoFS chain connection`

Reply:
> is it about what was fixed or after changes?

it always bothers me when i fill `Fixed`. i came to a mentioning a problem itself (so every point here describes _not_ expected things)

> `from now` / `no more` help

i can understand you but not sure we want to see hundreds of `from now` it this file. mb we need to find some common approach or try to paraphrase it. @roman-khimov

> and Morph client -> NeoFS chain connection

this is a common chain client

Reply:
fixed _smth_, like "RPC client failure when..."

Reply:
agree, fixed. but still think that we need to have some common behavior pattern here. an headache every time I need to expand this section even if I really want it to be clear